examples/flash_example/README.md

```diff
@@ -0,0 +1,30 @@
+# FLASH Federated Learning Example
+
+In this example, we implement the FLASH algorithm from the paper [FLASH: Fast and Robust Federated Learning Through Approximate Second-Order Method](https://proceedings.mlr.press/v202/panchal23a/panchal23a.pdf). This method enhances federated learning by addressing both statistical heterogeneity and concept drift issues.
```

Comment:
Perhaps I'm missing something, but it looks like the title (at least from the linked PDF) is "Flash: Concept Drift Adaptation in Federated Learning" rather than FLASH: Fast and Robust Federated Learning Through Approximate Second-Order Method?

Reply:
Changed, I have no idea where I got this name from 🤔